portfolio/src/main/webapp/index.html

```diff
@@ -1,26 +1,29 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>Alex's Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="myscript.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>Alex's Portfolio</h1>
-      <p>Hello! My name is Alexandre Walsh, but I go by Alex. 
-          I am a rising third-year at UCLA studying Computer Science and I'm from San Diego, CA. 
-          I was born in Barcelona, however, and, fun fact, I got to meet the president of Catalunya 
-          (Jordi Pujol) when I was 4 years old! I love exploring LA (like in the below picture, 
-          where I watched Caravan Palace perform at the Wiltern Theater) and was really looking forward to 
-          getting to explore the Mountain View area this summer, but am glad that I still get the amazing opportunity 
-          to work with Google and the Google Fi team this summer :)</p>
-    <div>
-      <img src="/images/Alex_at_the_Wiltern.jpg" style="width:432px;height:576px">
-    </div>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
+    <head>
+        <meta charset="UTF-8">
+        <title>Alex's Portfolio</title>
+        <link rel="stylesheet" href="style.css">
+        <script src="myscript.js"></script>
+    </head>
+    <body>
+        <div id="content">
+            <div id="left-text">
+                <h1>A Bit About Alex</h1>
+                <p>Hello! My name is Alexandre Walsh, but I go by Alex. 
+                I am a rising third-year at UCLA studying Computer Science and I'm from San Diego, CA. 
+                I was born in Barcelona, however, and, fun fact, I got to meet the president of Catalunya 
+                (Jordi Pujol) when I was 4 years old! I love exploring LA (like in the picture, 
+                where I watched Caravan Palace perform at the Wiltern Theater) and was really looking forward to 
+                getting to explore the Mountain View area this summer, but am glad that I still get the amazing opportunity 
+                to work with Google and the Google Fi team this summer :)
+                </p>
+            </div>
+            <div id="right-pic">
+                <img src="/images/Alex_at_the_Wiltern.jpg" style="width:432px;height:576px;vertical-align:middle">
```

Comment:
Try to use a class to include all the styles here to make the code cleaner. For example: https://www.w3schools.com/cssref/sel_class.asp

Reply:
i've been using id's instead of classes. should I use classes and get rid of the id's instead?